developer-docs/upgrade/build_n_deploy.md

```diff
@@ -7,11 +7,36 @@ allowSearch: true
 keywords: Upgrade, Sunbird 2.3.0, Core, elasticsearch
 ---
 
-## Overview
+**New load balancer setup for Swarm manager**
+
+(Optional - Refer the variable sunbird_swarm_manager_lb_ip in inventory for comments)
+
+
+To setup LB for Swarm managers, execute the following instructions for each of the mentioned fields:
```

Comment:
@keshavprasadms can you expand LB? Also, the information below is not a set of instructions. Either we need to write them as instructions or as a table, with clear header to explain what is meant by the information that follows the field name.

Reply:
@Sunitashelar This page is updated. Its assumed that the user has basic understanding on load balancer setup. A full blown explanation will require its own page. Also the same information is present here - http://qa.docs.sunbird.org/2.3.0/developer-docs/server-installation/load_balancer_setup/

If we are going to modify this, then the page in the link I shared also needs to be modified. Please take a call on this. We don't want to go deep into that details since this is azure specific. A lot of users might user AWS or GCP.

Reply:
@keshavprasadms just needed the acronym for LB expanded. No detailed explanation for load balancer is required.